c/CMakeLists.txt

```diff
@@ -168,6 +168,14 @@ else()
   BLAKE3_DISABLE_SIMD()
 endif()
 
+# cmake test support
+get_target_property(BLAKE3_SOURCES blake3 SOURCES)
+add_executable(blake3-testing ${BLAKE3_SOURCES} main.c)
+set_property(TARGET blake3-testing PROPERTY OUTPUT_NAME blake3)
+target_compile_definitions(blake3-testing PRIVATE BLAKE3_TESTING=1)
+enable_testing()
+add_test(test_vectors "${CMAKE_CURRENT_SOURCE_DIR}/test.py" WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
```

Comment:
I think we should `find_package(PYTHON)` and invoke the python executable with the script in order to allow the user to specify the python installation.